pubsub/azure/servicebus/message.go

```diff
@@ -138,6 +139,77 @@ func NewPubsubMessageFromASBMessage(asbMsg *azservicebus.ReceivedMessage, topic
 	return pubsubMsg, nil
 }
 
+func NewBulkMessageEntryFromASBMessage(asbMsg *azservicebus.ReceivedMessage) (pubsub.BulkMessageEntry, error) {
+	entryID, err := uuid.NewRandom()
+	if err != nil {
+		// This should never happen.
```

Comment:
Actually this can happen if the kernel doesn't have enough entropy :)

Reply:
TIL, I removed the comment! 😸